toast-tk-runtime/src/main/java/io/toast/tk/runtime/block/VariableBlockBuilder.java

```diff
@@ -3,38 +3,36 @@
 import java.util.List;
 
 import com.google.inject.Inject;
-import com.google.inject.Injector;
 
 import io.toast.tk.dao.domain.impl.test.block.VariableBlock;
 import io.toast.tk.dao.domain.impl.test.block.line.BlockLine;
 import io.toast.tk.runtime.IActionItemRepository;
-import io.toast.tk.runtime.block.IBlockRunner;
 
-public class VariableBlockBuilder implements IBlockRunner<VariableBlock>{
+public class VariableBlockBuilder implements IBlockRunner<VariableBlock> {
 
 	@Inject
-	IActionItemRepository objectRepository;
-	
+	private IActionItemRepository objectRepository;
+
 	@Override
 	public void run(final VariableBlock block) {
-		block.getBlockLines().stream().forEach(blockLine -> putUserVariable(blockLine));
+		block.getBlockLines().forEach(this::putUserVariable);
 	}
 
 	private void putUserVariable(final BlockLine blockLine) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Private method 'putUserVariable' is never used. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)